static/css/gni.css

```diff
@@ -322,10 +322,10 @@ body {
   font-size: 0.95rem;
   margin: 0 -24px;
   margin-left: 6px;
-  padding: 3px 24px 3px 6px;
+  padding: 0 24px 0 6px;
   border-top-right-radius: 6px;
   border-bottom-right-radius: 6px;
-  border-left: 4px solid #ccc;
+  /* border-left: 4px solid #ccc;  */
```

Comment:
Was this accidentally removed?

Reply:
This is removed deliberately, because otherwise there're two solid lines on the left for each level. I'm no sure if there's a better way to get rid of those lines.